tests/test_baseline.py

```diff
@@ -240,64 +182,87 @@ def test_model_sliced(self):
             assert torch.allclose(output_sliced, output[::2])
 
 
-class TestLayerbyLayer:
-    def test_layers_find_error(self):
+# class TestLayerbyLayer:
```

Comment:
Do you foresee us needing this in the future? i.e. is there a reason not to just delete this as opposed to commenting it out?

Reply:
Yes, Removed it as the purpose of the test is already solved.